.github/workflows/build.yaml

```diff
@@ -43,19 +43,19 @@ jobs:
         run: docker compose up -d
       - name: wait for kafka
         run: |
-          wget https://downloads.apache.org/kafka/3.2.3/kafka_2.13-3.2.3.tgz
-          tar -xzf kafka_2.13-3.2.3.tgz
-          ./kafka_2.13-3.2.3/bin/kafka-topics.sh --bootstrap-server localhost:9092 --list
-          ./kafka_2.13-3.2.3/bin/kafka-topics.sh --bootstrap-server localhost:9092 --list
+          wget https://downloads.apache.org/kafka/3.4.1/kafka_2.13-3.4.1.tgz
```

Comment:
Not sure, why this file is not synced from  `.github` though

Reply:
Looks like we are performing these extra steps specific to KP. To sync them from workflows, we'd have to copy over these to that repo, and sync from there. Doing that for a single repository doesn't seem worth the effort.

Reply:
In general i feel this file can very much be synced from .github and maybe use one of the existing ones. If we feel like we need separate steps for integration tests then we can use a build file without the int tests from .github and a custom integration_tests.yaml file.